tests/presenters/teaser-presenter.spec.js

```diff
@@ -12,6 +12,21 @@ describe('Teaser Presenter', () => {
 
 	context('classModifiers', () => {
 
+		it('produces an empty array if mods undefined', () => {
+			const content = { mods: undefined };
+			subject = new Presenter(content);
+			expect(subject.classModifiers).to.deep.equal([]);
+		});
+
+		it('does not save previous teaser modifiers to new teaser', () => {
+			const content = { isOpinion: true };
+			let teaser1 = new Presenter(content);
+			let teaser2 =new Presenter(content);
```

Comment:
I'll add this as a different test, because the condition we were trying to avoid isn't related to the mods that are external to the classModifiers — since we `Object.assign(context, options.hash)` in the nTeaserPresenter, the original `mods` get overridden. But we repeatedly push extra modifiers into the `Presenter` mods when we call `classModifiers`